scripts/performance-helper.psm1

```diff
@@ -360,7 +361,14 @@ function Invoke-LocalExe {
     $FullCommand = "$Exe $RunArgs"
     Write-Debug "Running Locally: $FullCommand"
 
-    return (Invoke-Expression $FullCommand) -join "`n"
+    $LocalJob = Invoke-Command -ScriptBlock { & $Exe ($RunArgs).Split(" ") } -AsJob
+
+    # Wait 60 seconds for the job to finish
+    Wait-Job -Job $LocalJob -Timeout 60 | Out-Null
```

Comment:
Timeout should be an (optional) input argument, all the way up to the user when it calls the script.

Reply:
The timeout would be per individual test, not the whole script. But that'd be fine.